Rename variable payments to output payment singular references (#41)
DFE-Digital/cpd-payment-calculations: README.md

> Per participant price £995 >>
per participant service fee £398 (40%) >> monthly service fee £27k >> total service fee £796k
>
> Per participant price £995 >> per participant output payment £597 (60%) >> per participant output payment for a retention period £119 (20% of 60%) >> output payment subtotal for a retention period with 1900 retained participants £226k

* "Participants" includes both teachers and mentors.

> * "Output" payments are payments made based on the performance of the training provider (i.e. their output). Previously known as "variable" in the code.

Reviewer: ```suggestion
* "Output payments" are payments made based on the performance of the training provider (i.e. their output). Previously known as "variable payments" in the code.
```

> * Note that the contractual term for "output payment" is actually "output payment", however we're avoiding this in the code to avoid the inevitable confusion with program input/output terminology. These are payments made based on the performance of the training provider (i.e. their output).

Reviewer 2: ```suggestion
* "Output" payments are payments made based on the performance of the training provider (i.e. their output). Previously known as "variable" in the code.
```
